packages/manager/modules/nutanix/src/dashboard/general-info/template.html

```diff
@@ -33,20 +31,24 @@
                 term="{{:: 'nutanix_dashboard_cluster_commercial_name' | translate }}"
             >
                 <oui-tile-description>
-                    <oui-spinner
-                        data-ng-if="$ctrl.loadingServicesDetails"
-                        data-size="s"
-                    ></oui-spinner>
-                    <span
-                        data-ng-if="!$ctrl.loadingServicesDetails && $ctrl.packType !== $ctrl.NUTANIX_PERSONAL_LICENSE_EDITION"
-                        data-ng-bind="$ctrl.servicesDetails.billing.plan.invoiceName"
+                    <protected-data
+                        is-hidden="!$ctrl.haveServicesAuthorizations()"
                     >
-                    </span>
-                    <span
-                        data-ng-if="!$ctrl.loadingServicesDetails && $ctrl.packType === $ctrl.NUTANIX_PERSONAL_LICENSE_EDITION"
-                        data-ng-bind="$ctrl.servicesDetails.billing.plan.invoiceName"
-                    >
-                    </span>
+                        <oui-spinner
+                            data-ng-if="$ctrl.loadingServicesDetails"
+                            data-size="s"
+                        ></oui-spinner>
+                        <span
+                            data-ng-if="!$ctrl.loadingServicesDetails && $ctrl.packType !== $ctrl.NUTANIX_PERSONAL_LICENSE_EDITION"
+                            data-ng-bind="$ctrl.servicesDetails.billing.plan.invoiceName"
+                        >
+                        </span>
+                        <span
+                            data-ng-if="!$ctrl.loadingServicesDetails && $ctrl.packType === $ctrl.NUTANIX_PERSONAL_LICENSE_EDITION"
```

Comment:
on affiche le même span que le packType soit égal à NUTANIX_PERSONAL_LICENSE_EDITION ou non ?

Reply:
No effectively I not read in detail the old code to focus my change. 

But I can improve it. Nice seeing